tests/t-kdf.c

```diff
@@ -1497,6 +1497,63 @@ check_argon2 (void)
     goto again;
 }
```

Comment:
https://lists.gnutls.org/pipermail/gcrypt-devel/2022-July/005346.html

Reply:
This looks correct.